commands/out.go

```diff
@@ -105,6 +105,84 @@ func (o *Out) Execute() error {
 		}
 
 		tagsToPush = append(tagsToPush, repo.Tag(tag))
+
+		if req.Params.BumpAliases && ver.Prerelease() == "" {
+			auth := &authn.Basic{
+				Username: req.Source.Username,
+				Password: req.Source.Password,
+			}
+
+			imageOpts := []remote.Option{}
+
+			if auth.Username != "" && auth.Password != "" {
+				imageOpts = append(imageOpts, remote.WithAuth(auth))
+			}
+
+			versions, err := remote.List(repo, imageOpts...)
```

Comment:
I don't think we want to account for registry mirrors; `put` doesn't use the mirror because it needs to make sure it's talking to the source of truth.